pysrc/bytewax/connectors/kafka/operators.py

```diff
@@ -91,27 +91,26 @@ def input(  # noqa A001
     as `bytewax.connectors.kafka.KafkaSourceMessage` objects.
 
     Args:
-        step_id:
-            Unique name for this step
-        flow:
-            A Dataflow
-        brokers:
-            List of `host:port` strings of Kafka brokers.
-        topics:
-            List of topics to consume from.
-        tail:
-            Whether to wait for new data on this topic when the
+        step_id: Unique name for this step
+
+        flow: A Dataflow
+
+        brokers: List of `host:port` strings of Kafka brokers.
+
+        topics: List of topics to consume from.
+
+        tail: Whether to wait for new data on this topic when the
```

Comment:
Is this a different formatter config, or just a style choice? I like it, but if it's an automated thing I might need to update something in my local environment